ambient.d.ts

```diff
@@ -246,18 +246,38 @@ declare module '@mojaloop/central-services-shared' {
       TP_CONSENT_PUT_ERROR: string;
     };
   }
+
+  enum EventTypeEnum {
+    AUTHORIZATION = 'AUTHORIZATION',
+    NOTIFICATION = 'NOTIFICATION',
+    TRANSACTION_REQUEST = 'TRANSACTION_REQUEST',
+  }
+
+  enum EventActionEnum {
+    EVENT = 'EVENT',
+    POST = 'POST',
+    PUT = 'PUT',
+  }
+
   interface Enum {
```

Comment:
that is lovely how source code is handling plurality: https://github.com/mojaloop/central-services-shared/blob/master/src/index.js#L4